Makefile.am

```diff
@@ -17,3 +21,10 @@ DIST_SUBDIRS = third_party lib include src generators targets CLI examples \
 bin tests frontends_extra $(MAYBE_PROTO)
 
 AM_DISTCHECK_CONFIGURE_FLAGS = --with-bmv2 --with-fe-cpp
+
+# ensure the distribution of the doxygen configuration file
+EXTRA_DIST = $(top_srcdir)/docs/doxygen.cfg \
+	     $(builddir)/doxygen-out/html $(builddir)/doxygen-out/pi.pdf
```

Comment:
what about `docs/bfn_latex.sty` and the md files, shouldn't they be included (if someone wants to rebuild the doc)?
you can't distribute the built files however, otherwise `make dist` fails. I suppose you could add a manual `dist` dependency on the doc, but I wouldn't do that. Instead we should have the Travis job build the documentation and upload it.